runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/OutputReceiverFactory.java

```diff
@@ -18,14 +18,16 @@
 
 package org.apache.beam.runners.fnexecution.control;
 
+import org.apache.beam.sdk.fn.data.FnDataReceiver;
+
 /**
  * A factory that can create output receivers during an executable stage.
  */
 public interface OutputReceiverFactory {
   /**
    * Get a new {@link RemoteOutputReceiver} for an output PCollection.
    *
-   * This call should only be invoked once per PCollectionId per factory.
+   * <p>This call should only be invoked once per PCollection id per factory.
```

Comment:
This restriction is only sort of true, as I would understand it. It's probably not strictly required, but some implementations may require this.

Reply:
It's not necessary in all cases, but this is something we want for runner implementors.